### PR TITLE
Adding Contributing information

### LINKED ISSUE #109: Add a section in the README about how to contribute with a new translation.
Explain how you can add a new translation to the plugin at the bottom of the README file.

### PR DESCRIPTION
**Pull request by jespino**
Fixes #109

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
+# Contributing to Mattermost Jitsi Plugin
```
Comment (jasonblais) [inline CONTRIBUTING.md:1]: When I tried to submit a PR with proposed suggestions for this file, it gave an error..so posting my suggestion below instead:

````
# Contributing to Mattermost Jitsi Plugin

Thank you for your interest in contributing! Join the [**Plugin: Jitsi**](https://community-daily.mattermost.com/core/channels/plugin-jitsi) channel on the Mattermost community server for discussion about this plugin.

## Reporting issues

If you think you found a bug, [please use the GitHub issue tracker](https://github.com/mattermost/mattermost-plugin-jitsi/issues/new?template=issue.md) to open an issue. To help us troubleshoot the issue, please provide the required information in the issue template.

## Translating strings

Mattermost Jitsi Plugin supports localization to various languages. We as maintainers rely on contributors to help with the translations.

The plugin uses [go-i18n](https://github.com/nicksnyder/go-i18n) as library and tool to manage translation. The CLI tool `goi18n` is required to manage translation. You can install it by running `env GO111MODULE=off go get -u github.com/nicksnyder/go-i18n/v2/goi18n`.

The localization process is defined below:
- During development, new translation strings may be added or existing ones updated.
- When a new version is planned to release soon, a repository maintainer opens an issue informing about the new release and mentions all translation maintainers in the issue.
- Translation maintainers submit PRs with new translations, which may get reviewed by other translators.
- After all translation PRs are merged, the new version is released. If a translation PR is not submitted within a week, the release is cut without it.

### Translation maintainers

- France: [@Extazx2](https://github.com/Extazx2)
- German: [@hanzei](https://github.com/hanzei)
- Spanish: [@jespino](https://github.com/jespino)

### Translate to a new language

Note: We use the German locale (`de`) in this example. When translating to a new language, replace `de` in the following commands with the locale you want to translate. [See available locales](https://github.com/mattermost/mattermost-server/tree/master/i18n).

1. Create a translation file:

   `touch asserts/i18n/translate.de.json`

2. Merge all current messages into your translation file:

   `make i18n-extract-server`

3. Translate all messages in `asserts/i18n/translate.de.json`.

4. Merge the translated messages into the active message files:

   `make i18n-merge-server`

5. Add your language to the list of [supported languages](https://github.com/mattermost/mattermost-plugin-jitsi#localization) in `README.md` and add yourself to the list of [translation maintainers](#translation-maintainers) in `CONTRIBUTING.md`.

6. [Submit a PR](https://github.com/mattermost/mattermost-plugin-jitsi/compare) with these files.

Once you've submitted a PR, your changes will be reviewed. Big thank you for your contribution!

### Translate existing languages

1. Ensure all translation messages are correctly extracted:

   `make i18n-extract-server`

2. Translate all messages in `asserts/i18n/translate.*.json` for the languages you are comfortable with.

3. Merge the translated messages into the active message files:

   `make i18n-merge-server`

4. Commit **only the language files you edited** and [submit a PR](https://github.com/mattermost/mattermost-plugin-jitsi/compare).

## Submitting bug fixes or features

We have [open help wanted issues](https://github.com/mattermost/mattermost-plugin-jitsi/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-desc+label%3A%22Help+Wanted%22) we are looking for help.

If you are contributing a feature, [please open a feature request](https://github.com/mattermost/mattermost-plugin-jitsi/issues/new?template=issue.md) first. This enables the feature to be discussed and fully specified before you start working on it. Small code changes can be submitted without opening an issue.

Note that this project uses [Go modules](https://github.com/golang/go/wiki/Modules). Be sure to locate the project outside of `$GOPATH`, or allow the use of Go modules within your `$GOPATH` with an `export GO111MODULE=on`.

## Development

This plugin contains both a server and webapp portion.

* Use `make dist` to build distributions of the plugin that you can upload to a Mattermost server.
* Use `make test` to run tests of the plugin.
* Use `make check-style` to check the style.
* Use `make deploy` to deploy the plugin to your Mattermost server. Before running make deploy you need to set a few environment variables:

```sh
export MM_SERVICESETTINGS_SITEURL=http://localhost:8065
export MM_ADMIN_USERNAME=admin
export MM_ADMIN_PASSWORD=password
```

* Use `make help` to know all useful targets for devleopment

````

File: README.md [hunk 2]
```diff
@@ -63,3 +73,9 @@ Inside the `/server` directory, you will find the Go files that make up the serv
 ### Web App
 
 Inside the `/webapp` directory, you will find the JS and React files that make up the client-side of the plugin. Within there, modify files and components as necessary. Test your syntax by running `npm run build`.
+
+## Contributing
+
+We welcome contributions for bug reports, issues, feature requests, feature implementations and pull requests. Feel free to [**file a new issue**](https://github.com/mattermost/mattermost-plugin-jitsi/issues/new/choose) or join the [**Plugin: Jitsi channel**](https://community.mattermost.com/core/channels/plugin-jitsi) on the Mattermost community server.
+
+For a complete guide on contributing to the plugin, see the [Contribution Guideline](CONTRIBUTING.md).
```
Comment (jasonblais) [inline README.md:81]:
```suggestion
For a complete guide on contributing to the plugin, see the [Contribution Guidelines](CONTRIBUTING.md).
```

File: README.md [hunk 3]
```diff
@@ -44,6 +44,16 @@ Go to **System Console > Plugins > Jitsi** and set the following values:
 
 You're all set! To test it, go to any Mattermost channel and click the video icon in the channel header to start a new Jitsi meeting.
 
+## Localization
```
Comment (jasonblais) [inline README.md:47]: Proposed updates:

```
### Localization

Mattermost Jitsi Plugin supports localization in multiple languages:
- English
- France
- German
- Spanish

The plugin automatically displays languages based on the following:
- For system messages, the locale set in **System Console > General > Localization > Default Server Language** is used.
- For user messages, such as help text and error messages, the locale set set in **Account Settings > Display > Language** is used.
```

cc @hanzei

Comment (jasonblais) [reply README.md:47]: Also, propose we move this under "Features", above "Installation"

**Comment by jasonblais** [comment]
@jespino I'm happy to make the proposed changes above prior to releasing v2.0

**Comment by jespino** [comment]
This is superseeded by the #139 PR

**Pull request closed without merging**